packages/@dcl/ecs/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@dcl/crdt": "file:../crdt",
     "@dcl/js-runtime": "file:../js-runtime",
-    "@dcl/protocol": "1.0.0-4075042272.commit-3585af9"
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4084048071.commit-d7cbc56.tgz"
```

Comment:
TODO: update to released protocol version

Reply:
done 👍